deposit-model/src/main/java/org/dataconservancy/nihms/model/DepositMetadata.java

```diff
@@ -274,16 +301,42 @@ public void setDoi(URI doi) {
         public String email;
 
         /**
-         * Principle Investigator (i.e. NIHMS "funding provider")
+         * Role for this entry for this person (multiple entries allowed per person)
          */
-        public boolean pi;
+        public PERSON_TYPE type;
 
-        /**
-         * NIHMS "Reviewer"
-         */
-        public boolean correspondingPi;
+        public Person() {}
 
-        public boolean author;
+        public Person(Person otherPerson) {
+            this.setFullName(otherPerson.getFullName());
+            this.setFirstName(otherPerson.getFirstName());
+            this.setMiddleName(otherPerson.getMiddleName());
+            this.setLastName(otherPerson.getLastName());
+            this.setType(otherPerson.getType());
+        }
+
+        // If "full name" has been set it will be returned,
+        // otherwise a combination of first/middle/last names will be returned.
+        public String getName() {
```

Comment:
@Trumbore So what is the difference between `getName()` and `getFullName()`?  Can the `*Name*` methods be Javadoc'd?